edsnlp/pipes/trainable/ner_crf/ner_crf.py

```diff
@@ -117,6 +118,33 @@ class TrainableNerCrf(TorchComponent[NERBatchOutput, NERBatchInput], BaseNERComp
     To train the model, refer to the [Training](/tutorials/training)
     tutorial.
 
+    Extensions
+    ----------
+    WARNING: THIS SECTION IS EXERIMENTAL AND MAY CHANGE!
+    The `eds.ner_crf` pipeline declares one extension on the `Span` object:
+
+    - the `span._.ner_prob`: The confidence score of the Named Entity Recognition
+    (NER) model for the given span.
+
+    The `ner_prob` is computed based on the Average Entity Confidence Score using
+    the following formula:
+
+    $$ \text{Average Entity Confidence Score} = \frac{1}{n}
+    \\sum_{i \\in \text{tokens}}1-p(O)_i
+    $$
+    Where:
+    - $n$ is the number of tokens.
+    - $\text{tokens}$ refers to the tokens within the span.
+    - $p(O)_i$ represents the probability of token $i$ belonging to class 'O'
+    (Outside entity).
+
+    By default, the confidence score is computed. However, if you don't need it, you
+    can disable its computation with:
+    ```{ .python }
+    nlp.pipes.ner.compute_prob = False
```

Comment:
By default, I'd rather not compute this score to prevent any performance degradation (although you've shown is below 5%)